mvn-build-all.bat

```diff
@@ -12,17 +12,28 @@ if not "%*" == "" (
 
 echo [INFO] Start a build.
 
+echo [DEBUG] Command arguments : "%DEFAULT_GOALS%"
+
+call mvn -U -f terasoluna-gfw-parent\pom.xml %DEFAULT_GOALS%
+        if not !ERRORLEVEL! == 0 (
+            echo [ERROR] Failed a build.
+            exit /B !ERRORLEVEL!
+        )
+
 echo [DEBUG] Command arguments : "%commandArgs%"
 
-for /f %%i in (mvn-build-all-targets.txt) do (
-    set pomFile=%%i\pom.xml
-    if exist !pomFile! (
-        call mvn -U -f !pomFile! %commandArgs%
+call mvn -U %commandArgs%
```

Comment:
@bttanakagn @ikeyat
In the first place, is `terasoluna-gfw-parent` is should be included in the root pom's module?